packages/host/app/components/operator-mode/stack.gts

```diff
@@ -0,0 +1,62 @@
+import Component from '@glimmer/component';
+import { Actions } from '@cardstack/runtime-common';
+import { StackItem } from '@cardstack/host/components/operator-mode/container';
+import OperatorModeStackItem from '@cardstack/host/components/operator-mode/stack-item';
+import { action } from '@ember/object';
+
+interface Signature {
+  Args: {
+    stackItems: StackItem[];
+    stackIndex: number;
+    publicAPI: Actions;
+    close: (stackItem: StackItem) => void;
+    cancel: (stackItem: StackItem) => void;
+    edit: (stackItem: StackItem) => void;
+    delete: (stackItem: StackItem) => void;
+    save: (stackItem: StackItem) => void;
+  };
+}
+
+export default class OperatorModeStack extends Component<Signature> {
+  @action
+  async dismissStackedCardsAbove(itemIndex: number) {
+    for (let i = this.args.stackItems.length - 1; i > itemIndex; i--) {
+      this.args.close(this.args.stackItems[i]);
+    }
+  }
+
+  <template>
+    {{! Argument of type 'unknown' is not assignable to parameter of type 'Element'. (this is because of ...attributes) }}
+    {{! @glint-ignore}}
```

Comment:
Can this be avoided by specifying the Element type as part of the signature?

Reply:
Good catch! I made the signature properly this time and was able to get rid of ignore statements.